addon.xml

```diff
@@ -2,17 +2,17 @@
 <addon id="plugin.video.netflix" name="Netflix" version="0.15.2" provider-name="libdev + jojo + asciidisco + caphm + castagnait">
   <requires>
     <import addon="xbmc.python" version="2.26.0"/>
-    <import addon="script.module.requests" version="2.12.4"/>
-    <import addon="script.module.pycryptodome" version="3.4.3"/>
-    <import addon="script.module.inputstreamhelper" version="0.3.3"/>
     <import addon="script.module.addon.signals" version="0.0.3"/>
+    <import addon="script.module.inputstreamhelper" version="0.3.3"/>
+    <import addon="script.module.pycryptodome" version="3.4.3"/>
+    <import addon="script.module.requests" version="2.12.4"/>
   </requires>
   <extension point="xbmc.python.pluginsource" library="addon.py">
     <provides>video</provides>
   </extension>
-  <extension point="xbmc.service" library="service.py" start="login" />
-  <extension point="xbmc.python.module" library="modules/enum" />
-  <extension point="xbmc.python.module" library="modules/mysql-connector-python" />
+  <extension point="xbmc.service" library="service.py"/>
```

Comment:
why have removed start at login?

Reply:
This is because kodi-addon-checker reports it as incorrect syntax (for Leia at least).
```
failed validating {'point': 'xbmc.service', 'library': 'service.py', 'start': 'login'} with XsdAttributeGroup(['point', 'id', 'name', 'library']):

Reason: 'start' attribute not allowed for element.

Schema:

  <xs:complexType xmlns:xs="http://www.w3.org/2001/XMLSchema">
    <xs:attribute name="point" type="xs:string" use="required" />
    <xs:attribute name="id" type="simpleIdentifier" />
    <xs:attribute name="name" type="xs:string" />
    <xs:attribute name="library" type="xs:string" use="required" />
  </xs:complexType>

Instance:

  <extension library="service.py" point="xbmc.service" start="login" />

Path: /extension

ERROR: Schema validation failed for the following points: xbmc.service 
```
So it is no longer needed/wanted.

Reply:
good